src/usdc-balance-change/usdc-balance-change.js

```diff
@@ -0,0 +1,167 @@
+const ethers = require('ethers');
+const BigNumber = require('bignumber.js');
+const {
+  Finding, FindingSeverity, FindingType, getJsonRpcUrl,
+} = require('forta-agent');
+
+// load agent config
+const config = require('../../agent-config.json');
+
+const initializeData = {};
+
+// load account addresses to monitor
+const accountAddresses = require('../../account-addresses.json');
+
+// load contract addresses to monitor
+const contractAddresses = require('../../contract-addresses.json');
+
+// provide ABI for USDC balanceOf()
+const usdcAbi = [
```

Comment:
Is there an ABI in a JSON file that could be copied into `forta-perpfi-suite/abis/`?

Alternatively, is there already an ABI there that matches the ABI for this (USDC)?
`BaseToken.json`?

Reply:
Switched to using IERC20Metadata.json from @perp/lushan, which provides the balanceOf() method.